spec/support/shared_examples/ca.rb

```diff
@@ -0,0 +1,96 @@
+shared_examples "a CA" do
+
+  it { is_expected.to respond_to(:create) }
+  it { is_expected.to respond_to(:create_keypair) }
+  it { is_expected.to respond_to(:create_keypair_in) }
+  it { is_expected.to respond_to(:reference_keypair) }
+  it { is_expected.to respond_to(:<=>) }
+
+  let :ca_name do
+    "some-ca"
+  end
+
+  let :bucket_name do
+    "a-bucket"
+  end
+
+  before do
+    @ca = described_class.create(ca_name, bucket_name)
+  end
+
+  describe ".create" do
+
+    it "should put the cert in s3" do
+      ca_cert = @ca.output["resource"]["aws_s3_bucket_object"].select { |_, obj|
+        obj[:key] == "/#{ca_name}/ca.cert" && obj[:bucket] == "#{bucket_name}"
```

Comment:
you don't need to interpolate here, `bucket_name` is already a string

Reply:
Search and replace gone interpolatey

Reply:
🕐 Timey Wimey Interpolatey 🕐